docs/source/index.rst

```diff
@@ -34,12 +34,13 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    torch.distributed <distributed>
    torch.legacy <legacy>
    cuda
+   bottleneck
+   checkpoint
```

Comment:
Does it display correctly with torch.utils as a prefix? Take that into account when sorting them

Reply:
It loads the `checkpoint.rst` file I believe :)

Reply:
I only sorted the torch.utils.* ones (bottleneck, checkpoint, ffi, cpp_extension, model_zoo, data)